Makefile

```diff
@@ -430,21 +430,17 @@ scorecard-build: custom-scorecard-tests
 	-f internal/images/custom-scorecard-tests/Dockerfile .
 
 # Local development/testing helpers
+ifneq ($(origin SAMPLE_APP_NAMESPACE), undefined)
+SAMPLE_APP_FLAGS += -n $(SAMPLE_APP_NAMESPACE)
+endif
 
 .PHONY: sample_app
-sample_app: sample_app_quarkus
+sample_app:
+	$(CLUSTER_CLIENT) apply $(SAMPLE_APP_FLAGS) -f config/samples/sample-app.yaml 
 
 .PHONY: undeploy_sample_app
-undeploy_sample_app: undeploy_sample_app_quarkus
-
-.PHONY: sample_app_quarkus
-sample_app_quarkus: undeploy_sample_app_quarkus
-	$(call new-sample-app,quay.io/andrewazores/quarkus-test:0.0.10)
-	$(CLUSTER_CLIENT) patch svc/quarkus-test -p '{"spec":{"$setElementOrder/ports":[{"port":9097},{"port":8080}],"ports":[{"name":"jfr-jmx","port":9097}]}}'
-
-.PHONY: undeploy_sample_app_quarkus
-undeploy_sample_app_quarkus:
-	- $(CLUSTER_CLIENT) delete all -l app=quarkus-test
+undeploy_sample_app:
+	$(CLUSTER_CLIENT) delete --ignore-not-found=$(ignore-not-found) -f config/samples/sample-app.yaml
```

Comment:
I think we need `$(SAMPLE_APP_FLAGS)` here as well to make sure we delete from the right namespace.

Reply:
oh yes! Missed that! Thanks!